src/views/pages/osscdn.html

```diff
@@ -5,7 +5,10 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<meta name="description" content="jsDelivr - A free, fast, and reliable Open Source CDN for npm and GitHub">
 		<title>OSSCDN.com is now jsDelivr</title>
-		<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:200,400,600,700">
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fontsource/open-sans@5.2.5/400.css">
```

Comment:
`Open Sans` doesn't have a weight of 200. The Google Fonts does not have as well.